none-attack/main.go

```diff
@@ -5,8 +5,8 @@ import (
 	"os"
 	"time"
 
-	jwt "github.com/KinakoExE/jwt-go"
-	"github.com/KinakoExE/jwt-go/request"
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/dgrijalva/jwt-go/request"
```

Comment:
[confirm]
It seems to work even if I rewrite it, but what is the reason for using your repository?

Reply:
`dgrijalva/jwt-go` does not allow us to use none algorithm without signature part but `KinakoExE/jwt-go` does.
But... now it's better to use `dgrijalva/jwt-go`, thank you.